plugins/module_utils/base.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import traceback
+
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from distutils.version import LooseVersion
+
+from ansible_collections.community.zabbix.plugins.module_utils.wrappers import ZapiWrapper
+
+
```

Comment:
How about add `zabbix_argment_spec` to standardization the arguments?  
For example something like the following.

```python
def zabbix_argment_spec():
    return dict(
        server_url=dict(type='str',
                        required=True,
                        aliases=['url']),
        login_user=dict(type='str',
                        required=True),
        login_password=dict(type='str',
                            required=True,
                            no_log=True),
        http_login_user=dict(type='str',
                             required=False,
                             default=None),
        http_login_password=dict(type='str',
                                 required=False,
                                 default=None,
                                 no_log=True),
        validate_certs=dict(type='bool',
                            required=False,
                            default=True)
    )
```

Reply:
You mean for the default arguments present for all modules? like it!
but i would put this in helpers.py as its not for the classes but before.

Reply:
> You mean for the default arguments present for all modules? like it!

Yes, that's right :)

> but i would put this in helpers.py as its not for the classes but before.

I think it's good.

Reply:
leave that to followup PR please

Reply:
This was proposed in #120 by @tcraxs , so no need for followup request. Once that is merged, we should need to adjust other modules as well.